src/eddymotion/model/base.py

```diff
@@ -521,6 +522,72 @@ class DKIModel(BaseDWIModel):
     _model_class = "dipy.reconst.dki.DiffusionKurtosisModel"
 
 
+class GaussianProcessModel(BaseModel):
+    """A Gaussian process model for DWI data based on [Andersson15]_."""
+
+    __slots__ = (
+        "_dwi",
```

Comment:
```suggestion
        "_gtab",
```

Reply:
This is now deprecated after https://github.com/jhlegarreta/eddymotion/pull/7, neither the signal nor the gradient table are slots of the class in the new signature, so resolving. We'll see how the new framework comes into life after we stabilize things.